pipenv/routines/check.py

```diff
@@ -109,32 +104,100 @@ def get_requirements(project, use_installed, categories):
 
 
 def create_temp_requirements(project, requirements):
-    temp_requirements = tempfile.NamedTemporaryFile(
-        mode="w+",
-        prefix=f"{project.virtualenv_name}",
-        suffix="_requirements.txt",
-        delete=False,
+    """Create a temporary requirements file that safety can access."""
```

Comment:
This step is not necessary as safety can directly work with pipfiles.
We can remove it in the future.